### ISSUE TITLE
JSBSim model for the ARDrone2 not accurate enough

### ISSUE THREAD

**Issue by OpenUAS** [paparazzi/paparazzi#1039]
So fare this is the list of JSBSim models that need improvement

- [ ] ARDrone2
- [ ] Bebop
- [ ] Bebop 2
- [ ] Disco

Example of test to perform

Performing a default NPS sim session and try to circle CAM point results in crashed ARdrone2.

Start NPS, Flightplan Start Engine, Takeoff ,Go P1, wait a bit then flightplan circle CAM.
I tried this a day or 3? ago and  could happily circle multiple times around but then exiting to P2 made drone fly away fast and wide and crash.

Tried differend entries point P1, P2 waypoint heights etc. all non working.

Basically we need a JSBSim model for the AC's that better match the real thing.

##

**Comment by OpenUAS** [comment]
Relate info pages
* http://wiki.paparazziuav.org/wiki/AR_Drone_2/JSBSim
* http://wiki.paparazziuav.org/wiki/AR_Drone_2/JSBSIM_.xml_Configuration_File

**Comment by OpenUAS** [comment]
Found a student that wants to try and finish it

**Comment by gautierhattenberger** [comment]
NPS simulation have been improved with a better timing of the sensors #2769
